srv/salt/_modules/advise.py

```diff
@@ -29,6 +29,14 @@ def reboot(running, installed):
 
     return True
 
+def generic(message):
+    """
+    Used to print arbitrary text to the screen.
+    """
+    message = message
+    log.info(message)
 
+    proc = Popen([ "/usr/bin/wall" ], stdin=PIPE, stdout=PIPE, stderr=PIPE)
+    output = proc.communicate(input=message)
```

Comment:
`proc.communicate(input=message)`

with message being one of [None, False, ""]

`proc.returncode` is always 0

I'll get rid of the assignment